packages/web/src/features/translation/TranslationSidebar.tsx

```diff
@@ -104,7 +150,37 @@ export const TranslationSidebar = ({
                 </div>
               )}
             </Tab.Panel>
-            <Tab.Panel>{t('common:coming_soon')}</Tab.Panel>
+            <Tab.Panel>
+              <div className="flex flex-col gap-2 pb-2">
+                <h2 className="font-bold">{t('translate:translator_notes')}</h2>
+                {translatorNote?.authorName && (
+                  <span className="italic">
+                    {t('translate:note_description', {
+                      timestamp: translatorNote?.timestamp
+                        ? new Date(translatorNote?.timestamp).toLocaleString()
+                        : '',
+                      authorName: translatorNote?.authorName ?? '',
+                    })}
+                  </span>
+                )}
+                {canEditNote ? (
+                  <RichTextInput
+                    key={word.id}
+                    name="noteContent"
+                    value={noteContent}
+                    onBlur={async (e) => {
+                      saveNote(e.target.value);
+                      saveNote.flush();
+                    }}
+                    onChange={async (e) => {
+                      saveNote(e.target.value);
+                    }}
+                  />
+                ) : (
+                  <RichText content={noteContent} />
+                )}
```

Comment:
One thing I had forgotten about is that translator's notes aren't accessible if you aren't a member of the language or a platform admin. So we should only show the header, timestamp and note if you have read access, and then if you have additional edit access (because you are a translator), you get the rich text input.

Reply:
done